data-collection/deploy/account-collector.yaml

```diff
@@ -190,6 +220,8 @@ Resources:
           ROLE_NAME: !Ref ManagementRoleName
           MANAGEMENT_ACCOUNT_IDS: !Ref ManagementAccountID
           RESOURCE_PREFIX: !Ref ResourcePrefix
+          BUCKET_NAME: !Ref DestinationBucket
+          ACCOUNT_LIST_KEY: "account-list/account-list.json"
```

Comment:
json is quite hard to generate from customer side. Maybe let's use csv?
also consider setting the path as Lambda env variable so it could be adjusted by customers if needed?

Reply:
I have modified the solution to allow for either CSV or JSON. They can just drop either into the S3 folder and it will get picked up.

The the path is defined as a Lambda variable already so they can adjust it